e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -0,0 +1,70 @@
+import TextOptions from './text-options';
+
+function textXpathHelper(opts: TextOptions) {
+  let txt;
```

Comment:
nit: this local var isn't really needed, you could just return directly inline

Reply:
👍